docs.md

```diff
@@ -0,0 +1,98 @@
+# core/v1 
+ 
+## Namespace 
+
+### Validation Checks
+Note: The kube-system namespace, unlike other namespaces, has a "failPolicy" of "ignore" on update calls.
+
+#### PSA Label Validation
+Validates that users who are changing a label for PSA enforcement (on update, or creating a namespace with a label for PSA enforcement in create) have the `updatepsa` verb on `projects` in `management.cattle.io/v3`. See the [upstream docs](https://kubernetes.io/docs/concepts/security/pod-security-admission/) for more information on the effect of these labels.
```

Comment:
I'm a little uneasy about this change since I think the wording is less precise - i.e. if you are creating a namespace with this annotation we still check this value. The enforcement is on the namespace object, not on the label.

That being said, I think it's a lot easier to read.

Reply:
Change made.